apps/Minecraft Java Server/install

```diff
@@ -0,0 +1,203 @@
+        sudo apt update
+        ;;
+    *)
+        error "No java version selected, exiting the script"
+        ;;
+esac
+
+case "$arch" in
+    "64")
+        case "$java_selection" in
+            "Java 8")
+                (install_packages temurin-8-jdk)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install temurin packages"
+                fi
+                java_location="/usr/lib/jvm/java-8-temurin-arm64/bin/java"
+                ;;
+            "Java 16")
+                (install_packages adoptopenjdk-16-hotspot-jre)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install adoptopenjdk packages"
+                fi
+                java_location="/usr/lib/jvm/adoptopenjdk-16-hotspot-jre-arm64/bin/java"
+                ;;
+            "Java 17")
+                (install_packages temurin-17-jdk)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install temurin packages"
+                fi
+                java_location="/usr/lib/jvm/java-17-temurin-arm64/bin/java"
+                ;;
+        esac
+        ;;
+    "32")
+        case "$java_selection" in
+            "Java 8")
+                (install_packages temurin-8-jdk)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install temurin packages"
+                fi
+                java_location="/usr/lib/jvm/java-8-temurin-armhf/bin/java"
+                ;;
+            "Java 16")
+                (install_packages adoptopenjdk-16-hotspot-jre)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install adoptopenjdk packages"
+                fi
+                java_location="/usr/lib/jvm/adoptopenjdk-16-hotspot-jre-armhf/bin/java"
+                ;;
+            "Java 17")
+                (install_packages temurin-17-jdk)
+                if [ $? != 0 ];then
+                    sudo rm -f /etc/apt/sources.list.d/adoptium.list /usr/share/keyrings/adoptium-archive-keyring.gpg
+                    error "Failed to install temurin packages"
+                fi
+                java_location="/usr/lib/jvm/java-17-temurin-armhf/bin/java"
+                ;;
+        esac
+        ;;
+    *)
+        error "arch variable is not set, can not continue"
```

Comment:
error "arch variable is not set, can not continue" should be
  error "Arch variable is not set, can not continue".

Reply:
I left it lowercase because the variable name is lowercase

Reply:
We could add a period, though..